src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -1216,6 +1218,30 @@ public void tempDirVariable() {
         });
     }
 
+    @Test
+    @Issue("JENKINS-63486")
+    public void getOwnerTaskPermissions() {
+        story.then(r -> {
+            MockFolder f = r.createFolder("f");
+            WorkflowJob p = f.createProject(WorkflowJob.class, "p");
+            p.setDefinition(new CpsFlowDefinition("node() { semaphore('wait') }", true));
+
+            r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
+            r.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy()
+                    .grant(Job.DISCOVER).onFolders(f).toEveryone()
+                    .grant(Job.READ).onItems(p).toEveryone());
+            User alice = User.get("alice", true, Collections.emptyMap());
+
+            WorkflowRun b = p.scheduleBuild2(0).waitForStart();
+            SemaphoreStep.waitForStart("wait/1", b);
+            for (Executor e : Jenkins.get().toComputer().getExecutors()) {
```

Comment:
There is probably a direct way to get the executor running the `node` step, but I could not think of it offhand.